packages/client/src/pages/Home.tsx

```diff
@@ -324,53 +344,70 @@ export const Home = (): JSX.Element => {
           </p>
         </div>
       )}
-
-      <h1 className="bg-clip-text bg-gradient-to-r font-bold from-purple-400 mb-6 mt-20 text-center text-transparent text-4xl to-pink-400 via-cyan-400">
+      <div className="flex justify-center my-6">
```

Comment:
@thebeyondr since there's no real pirate theme in the game, could we actually lose the animated flag?